File: src/main/java/com/GoldenMine/parser/pronoun/IPronoun것.java
package com.GoldenMine.parser.pronoun;

import com.GoldenMine.parser.Sentence;
import com.GoldenMine.parser.Variable;
import com.GoldenMine.parser.VariableStorage;

public class IPronoun것 implements IPronoun {

    @Override
    public String getType() {
        return "것";
    }

    @Override
    public Variable perform(Sentence sentence, VariableStorage storage) {
        return sentence.get서술어().perform(sentence, storage);
    }
}

File: src/main/java/kr/goldenmine/parser/pronoun/IPronoun것.kt
package kr.goldenmine.parser.pronoun

import kr.goldenmine.parser.Sentence
import kr.goldenmine.parser.Variable
import kr.goldenmine.parser.VariableStorage

class IPronoun것 : IPronoun {

    override val type: String
        get() = "것"

    override fun perform(sentence: Sentence, metadata: List<Any>?, storage: VariableStorage): Variable? {
        return sentence.서술어.perform(sentence, metadata, storage)
    }
}
